packages/react-components/src/components/Loader/Loader.module.scss

```diff
@@ -14,6 +14,11 @@
   flex-direction: row;
   flex-wrap: nowrap;
 
+  &--cover {
```

Comment:
Is this naming convention fine for @martagranisz?

Reply:
It's removed already